### ISSUE TITLE
pycbc_make_inference_workflow: confusing --gps-end-time option

### ISSUE THREAD

**Issue by dbkeitel** [gwastro/pycbc#2742]
Another commandline option @spxiwh and I found quite confusing: 
```
# analysis time option
# only required if not using input workflow file options
parser.add_argument("--gps-end-time", type=float, nargs="+", default=None,
                    help="Times to analyze. If given workflow files then "
"this option is ignored.")
```

1. From reading the code, we suspect (but aren't quite sure) this is "end-time" not in the sense of "last bit of data to read", but in the sense of "template end time"="trigger time" - anyone can confirm? If so, it should be renamed to avoid confusing with the other use of the word, which dominates in ini files etc. Otherwise, we'd be even more confused why there isn't a --gps-start-time at the same interface level.
1. The help talks about "workflow files" but we couldn't figure out which exact option in which exact file would override this; in fact it seems to look at the "bank_file", "statmap_file" and "single_detector_triggers" but not the workflow ini file; this should be clarified to the user.

**Comment by dbkeitel** [comment]
@cdcapano @cmbiwer any opinions on this? Would it be ok for me to make a PR renaming this to, say, "--gps-trigger-time"?

**Comment by cdcapano** [comment]
Fixed by #3140.
